paper.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 'SlicerITKUltrasound: A 3D Slicer extension for ultrasound image formation, processing, and analysis'
+tags:
+  - ultrasound
+  - medical imaging
+  - 3D Slicer
+  - ITK
+authors:
+ - name: Matthew M McCormick
+   orcid: 0000-0001-9475-3756
+   affiliation: 1
+ - name: Mark L Palmeri
+   orcid: 0000-0001-8995-0050
+   affiliation: 2
+ - name: Jean-Christophe Fillion-Robin
+   affiliation: 1
+ - name: Stephen Aylward
+   affiliation: 1
+affiliations:
+ - name: Kitware, Inc.
+   index: 1
+ - name: Duke University
+   index: 2
+date: 15 December 2016
+bibliography: paper.bib
+---
+
+# Summary
+
+A 3D Slicer [@Fedorov2012] extension for ultrasound image formation,
+processing, and analysis. Interfaces built off the ITKUltrasound library
+[@McCormick2014]. The modules available in this extension are useful for both
+2D and 3D image generation for traditional B-mode imaging [@McCormick2010],
+but also next-generation ultrasound image modalities like ultrasound
+spectroscopy [@Aylward2016], acoustic radiation force imaging (ARFI)
+[@Palmeri2016], acoustic radiation force shear wave imaging (ARFI-SWEI), and
```

Comment:
Good idea.  Added suggested reference. We could use another too if @mlp6 prefers.